internal/daemonset/daemonset_test.go

```diff
@@ -26,7 +25,7 @@ const (
 )
 
 var _ = Describe("SetDriverContainerAsDesired", func() {
-	dg := controllers.NewDaemonSetCreator(nil, kernelLabel, scheme)
```

Comment:
If we now have a `daemonset` package, maybe this could be renamed as `daemonset.NewCreator`?

Reply:
fixed